Tasks/DownloadPackageV1/universal.ts

```diff
@@ -1,9 +1,9 @@
 import { IExecSyncResult, IExecOptions } from "azure-pipelines-task-lib/toolrunner";
-import * as artifactToolRunner from "packaging-common/universal/ArtifactToolRunner";
+import * as artifactToolRunner from "azure-pipelines-tasks-packaging-common/universal/ArtifactToolRunner";
 import * as tl from "azure-pipelines-task-lib";
```

Comment:
Could you please import this before other dependencies. There's an issue with task-lib which appears in case when a dependency requires another task-lib version rather than a task do and this dependency is imported before task-lib itself. This can lead to some unexpected errors if there's any version mismatch in future.

Reply:
Fixed